src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -60,6 +61,25 @@ public static boolean isBamFileName(final String inputFileName) {
         return BamFileIoUtils.BAM_FILE_EXTENSION.equalsIgnoreCase("." + FilenameUtils.getExtension(inputFileName));
     }
 
+    /**
+     * Given a Path, determine if it is an HDF5 file without requiring that we're on a platform that supports
+     * HDF5 (let the caller decide if a return value of false is fatal).
+     *
+     * @param hdf5Candidate a Path representing the input to be inspected
+     * @return true if the candidate Path is an HDF5 file, otherwise false
+     */
+    public static boolean isHDF5File(final Path hdf5Candidate) {
```

Comment:
I think it's fine here, but have no strong feelings.  Thanks again for doing this, @cmnbroad!

Reply:
It would make sense to add to the hdf5 bindings in a safe way, but updating that library is a pain.  There's an accumulating list of reasons we should update it though, so we should at some point revise it and move this there I think.